lxd/instance.go

```diff
@@ -126,6 +126,7 @@ func instanceCreateFromImage(d *Daemon, args db.InstanceArgs, hash string, op *o
 	if img.Properties != nil {
 		for k, v := range img.Properties {
 			args.Config[fmt.Sprintf("image.%s", k)] = v
+			logger.Errorf("Setting: image.%v=%v", k, v)
```

Comment:
@stgraber should this be Errorf?

Reply:
Doh, this is leftover debugging, let me drop it.

Reply:
Fixed